cmd/skopeo/copy.go

```diff
@@ -99,6 +99,16 @@ var copyCmd = cli.Command{
 			Usage: "Use `USERNAME[:PASSWORD]` for accessing the destination registry",
 		},
 		cli.StringFlag{
+			Name:  "src-no-creds",
+			Value: "",
+			Usage: "Access the source registry anonymously",
+		},
+		cli.StringFlag{
+			Name:  "dest-no-creds",
+			Value: "",
+			Usage: "Access the destination registry anonymously",
+		},
+		cli.StringFlag{
```

Comment:
note to self.  We really should uppercase the first word in all Usage messages, or lower case them all.